src/test/kotlin/three/consulting/epoc/utils/FileReader.kt

```diff
@@ -0,0 +1,9 @@
+package three.consulting.epoc.utils
+
+import java.io.File
+import java.nio.charset.Charset
+
+fun jsonReader(location: String): String {
+    return File(location).readText(Charset.defaultCharset())
+}
```

Comment:
Nice. This would also be a prime spot for creating a unit test, but that needs to be done with a different test runner than the current one so we'll circle back to this later on.